docs/Installation.md

```diff
@@ -133,7 +133,7 @@ sudo sed -i '/\bengine\b/ s/=.*/= MySQL/' /etc/zonemaster/backend_config.ini
 Install, configure and start database engine:
 
 ```sh
-sudo yum install mariadb-server
+sudo yum -y install mariadb-server

```

Comment:
Interactive and non-interactive, that is the question. With this change it will be different for different OSs.